pycbc/io/live.py

```diff
@@ -22,30 +22,34 @@ class CandidateForGraceDB(object):
     """This class provides an interface for uploading candidates to GraceDB.
     """
 
-    def __init__(self, ifos, coinc_results, **kwargs):
+    def __init__(self, coinc_ifos, ifos, coinc_results, **kwargs):
         """Initialize a representation of a zerolag candidate for upload to
         GraceDB.
 
         Parameters
         ----------
+        coinc_ifos: list of strs
+            A list of the originally triggered ifos with SNR above threshold
+            for this candidate, before possible significance followups.
         ifos: list of strs
-            A list of the ifos participating in this candidate.
+            A list of the ifos with triggers identified in coinc_results for
+            this candidate: ifos contributing to significance
         coinc_results: dict of values
             A dictionary of values. The format is defined in
             `pycbc/events/coinc.py` and matches the on-disk representation in
             the hdf file for this time.
         psds: dict of FrequencySeries
-            Dictionary providing PSD estimates for all involved detectors.
+            Dictionary providing PSD estimates for all detectors observing.
         low_frequency_cutoff: float
             Minimum valid frequency for the PSD estimates.
         high_frequency_cutoff: float, optional
             Maximum frequency considered for the PSD estimates. Default None.
-        followup_data: dict of dicts, optional
+        skyloc_data: dict of dicts, optional
```

Comment:
"skyloc" is not a particularly better name than "followup" here, because BAYESTAR does more than *sky* localization (full 3D spatial localization + Bayes factors). Maybe I should have just chosen "bayestar_data" in the beginning :)

Reply:
And yes, the docstring below is also not completely correct because of the same point.

Reply:
The intention here is not to give a complete description of bayestar, it is to resolve the ambiguity of 'followup' now that we have significance followup as well